lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/IdentityVerificationService.java

```diff
@@ -91,7 +91,7 @@ public DocumentCheckVerificationResult verifyIdentity(DrivingPermitForm drivingP
                     result.setSuccess(documentCheckResult.isExecutedSuccessfully());
 
                 } else {
-                    LOGGER.warn("Fraud check failed");
+                    LOGGER.warn("Document check failed");
```

Comment:
Only small but might make more sense to specify it's a driving licence incase theres confusion with passport or other documents that could be brought in down the line